src/components/dataset/schemaCreation/DatasetSchemaInformationView.tsx

```diff
@@ -0,0 +1,334 @@
+import React, { useMemo, useState } from 'react';
+import ReactDOMServer from 'react-dom/server';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+import { WithStyles, withStyles } from '@mui/styles';
+import {
+  Typography,
+  TextField,
+  CustomTheme,
+  Grid,
+  Select,
+  MenuItem,
+  Autocomplete,
+} from '@mui/material';
+import { clsx } from 'clsx';
+import { TdrState } from 'reducers';
+import { Controller, useFormContext } from 'react-hook-form';
+import SimpleMDE from 'easymde';
+import { SimpleMdeReact } from 'react-simplemde-editor';
+import { GCP_REGIONS, AZURE_REGIONS } from 'constants/index';
+import isEmail from 'validator/lib/isEmail';
+import WithoutStylesMarkdownContent from '../../common/WithoutStylesMarkdownContent';
+
+const styles = (theme: CustomTheme) => ({
+  contentContainer: {
+    marginTop: '1rem',
+    maxWidth: 1000,
+  },
+  form: {
+    width: '100%',
+  },
+  formFieldContainer: {
+    minHeight: 135,
+  },
+  formFieldDescription: {
+    marginBottom: 10,
+  },
+  formLabel: {
+    display: 'block',
+    fontWeight: 'bold',
+    marginTop: 10,
+    marginBottom: 10,
+  },
+  formLabelError: {
+    color: theme.palette.error.main,
+  },
+  formInput: {
+    width: '100%',
+  },
+  formInputError: {
+    color: theme.palette.error.main,
+    fontSize: '0.75rem',
+    lineHeight: '1.66',
+    marginLeft: 14,
+  },
+});
+
+interface IProps extends WithStyles<typeof styles> {
+  userEmail: string;
+}
+
+const DatasetSchemaInformationView = withStyles(styles)(({ classes }: IProps) => {
+  const [regionOptions, setRegionOptions] = useState(GCP_REGIONS);
+  const {
+    register,
+    control,
+    formState: { errors },
+  } = useFormContext();
+
+  const editorOptions = useMemo(
+    () =>
+      ({
+        previewRender(markdownText) {
+          return ReactDOMServer.renderToString(
+            <WithoutStylesMarkdownContent markdownText={markdownText} />,
+          );
+        },
+        status: false,
+      } as SimpleMDE.Options),
+    [],
+  );
+
+  return (
+    <Grid container rowSpacing={2} columnSpacing={5} className={classes.contentContainer}>
```

Comment:
Tabbing through, it looks okay to me!

There is one area that sucks for keyboard navigation though, the description markdown editor treats the tabs as characters. Not sure how to resolve that right now, but I will add this as a bug ticket!

Otherwise, it seems okay!